src/main/java/org/terasology/splash/glfw/widgets/Widget.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.splash.glfw.widgets;
+
+import org.terasology.splash.glfw.graphics.Renderer;
+
+public interface Widget {
+    void render(Renderer renderer);
+
+    default void update(double dt) {
+    }
+
+    ;
```

Comment:
Stray semicolon - run the formatter on this file?

Reply:
Fixed!